commands/version/command.js

```diff
@@ -201,6 +207,12 @@ exports.builder = (yargs, composed) => {
         delete argv["skip-git"];
         log.warn("deprecated", "--skip-git has been replaced by --no-git-tag-version --no-push");
       }
+
+      if (argv.release) {
+        argv.createRelease = "github";
+        delete argv.githubReleases;
```

Comment:
```suggestion
        delete argv.githubRelease;
```

Reply:
and the deprecation message below, `release` -> `github-release`

Reply:
yes, we should not be introducing `--release`, as we're renaming `--github-release` -> `--create-release <type>`